<ui version="4.0" >
 <author>Olivier Goffart</author>
 <class>TextEffectPrefs</class>
 <widget class="QWidget" name="TextEffectPrefs" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>529</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="TabWidget3" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Colors</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox1" >
         <property name="title" >
          <string>Colors</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>11</number>
          </property>
          <property name="topMargin" >
           <number>11</number>
          </property>
          <property name="rightMargin" >
           <number>11</number>
          </property>
          <property name="bottomMargin" >
           <number>11</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item rowspan="5" row="0" column="0" >
           <widget class="QListWidget" native="1" name="mColorsListBox" />
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="mColorsAdd" >
            <property name="text" >
             <string>&amp;Add...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="mColorsRemove" >
            <property name="text" >
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="mColorsUp" >
            <property name="text" >
             <string>Move &amp;Up</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QPushButton" name="mColorsDown" >
            <property name="text" >
             <string>Move &amp;Down</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>21</width>
              <height>81</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="m_colorRandom" >
         <property name="text" >
          <string>Random order</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="Line" name="Line1" >
         <property name="frameShape" >
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="m_fg" >
         <property name="text" >
          <string>Change global text foreground color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QCheckBox" name="m_char" >
         <property name="text" >
          <string>Change color every letter</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QCheckBox" name="m_words" >
         <property name="text" >
          <string>Change color every word</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Effects</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="m_lamer" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>L4m3r t4lk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_casewaves" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>CasE wAVes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>279</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
 </customwidgets>
 <tabstops>
  <tabstop>TabWidget3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
